src/generated/rpg_animationframe.h

```diff
@@ -14,7 +14,9 @@
 
 // Headers
 #include <vector>
+#include <array>
 #include "rpg_animationcelldata.h"
+#include <array>
```

Comment:
too many <array> includes, iirc you have to handle thsi in "struct_headers" function